caas/kubernetes/provider/bootstrap.go

```diff
@@ -301,6 +301,27 @@ func (c *controllerStack) Deploy() (err error) {
 		}
 	}()
 
+	operatorImages, err := c.pcfg.GetControllerImagePath()
+	if err != nil {
+		return errors.Annotate(err, "failed to get operator image path")
+	}
+	selectedOperatorImage := ""
+	for _, img := range operatorImages {
+		logger.Debugf("prepulling operator image %s", img)
+		err = c.broker.operatorImagePrepullCheck(img)
+		if errors.IsNotFound(err) {
+			logger.Debugf("operator image %s not found", img)
+			continue
+		} else if err != nil {
+			return errors.Annotate(err, "operator image prepull check")
+		}
+		selectedOperatorImage = img
+		break
+	}
+	if selectedOperatorImage == "" {
+		return errors.Errorf("could not find suitable operator image")
```

Comment:
could we include the juju version of the image we are looking for in the error message